spring-boot/admin/src/main/resources/application.yml

```diff
@@ -3,29 +3,101 @@
 
 mvn.pom.project.version: @@project.version@@
 
-# user and password are expected to be for a gmail account
-ygg.mail.verification.server.url: http://localhost:8080
-
 logging:
   level:
     org.springframework.security: INFO
     nl.corizon.ygg: DEBUG
 
 # Necessary in order to run offline     
-ygg.admin.database.insertSampleDataOnStartup: true
-ygg.admin.gettingStarted.consumer.testService.id: 1
-ygg.admin.gitService.url: http://localhost
-ygg.admin.gitService.user: user
-ygg.admin.gitService.password: password
-ygg.admin.api-base-url-for-gateway: http://localhost
-ygg.proxy.executable-jar.path: path
-ygg.proxy.executable-jar.filename-original-prefix: prefix
-ygg.proxy.executable-jar.filename-custom.prefix: prefix
-ygg.proxy.executable-jar.filename-custom.suffix: suffix
-ygg.proxy.generate-bootstrap-properties.spring.application.name.prefix: prefix
-ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri: http://localhost
+ygg:
+   mail:
+      server:
+         url: "smtp.serviciodecorreo.es"
+         port: 465
+      sender:
+         from: "coatrack@kyoe.es"
+         user: ""
+         password: ""
+      verification.server.url: http://localhost:8080
+
+   admin:
+      database.insertSampleDataOnStartup: true
+      server:
+         url: http://localhost:8080
+      servicecovers:
+         path: /tmp
+         url: http://localhost/coatrack-servicecovers/
+      payment.commision:
+         variable: 50
+         fix: 1
+         bookKeeping.contact: fperezduran@bebr.nl
+      call-request-fee: 0
+    
+      gettingStarted.consumer.testService.id: 1
+      gitService:
+          url: http://localhost
```

Comment:
This breaks the generation of gateways, it only works with a real Git URL. I assume this was done on purpose, but it should be explained in a comment so that the reader knows what to change.

Maybe like this?
```
      gitService:
          # URL and credentials for the Git repository where CoatRack should store the config files for generated gateways
          url: https://url-of-git-repository:port
          user: username-for-git-repository
          password: password-for-git-repository